src/renderer/components/windows-title-bar.tsx

```diff
@@ -128,7 +128,6 @@ export default class WindowsTitleBar extends React.Component<{}, IState> {
             onClick={this.eventHandlers.onMinimize}
             onContextMenu={this.eventHandlers.onDisableContextMenu}
             onMouseDown={this.handleMouseDown}
-            disabled={isDisabled}
           >
             <svg x='0px' y='0px' viewBox='0 0 14 1'>
               <rect fill='rgba(255, 255, 255, 0.9)' width='14' height='0.6' />
```

Comment:
IMO it would be preferable to change svg color (maybe something like rgba(149, 149, 149, 0.9) rather than background color + disable button hovering